pinry-spa/src/components/PHeader.vue

```diff
@@ -109,6 +109,11 @@
                 </a>
               </div>
             </div>
+            <div class="locale-changer">
```

Comment:
We may add some auto-detection for locale, for example language settings in browser.

And, we could save the locale in local-storage/cookies.

Reply:
fair, I will then remove this locale-changer from the PHeader.vue since this originally for me to verify that the i18n was working. maybe look at storing locale in local-storage/cookies or auto-detection for locale in a PR.

Reply:
It's very nice to have an explicit “locale changer” and it should have highest priority than any "auto-detection".

We'd better save it once user select "locale" by hand.

For example, an user could use "Chinese" even he use an browser which prefer "English" (it's me hahahahaha).

Reply:
sorry, just read over the response now and have added the locale changer back into the PR. will have a look at auto-detection when I have time, seems like we will need to make utility that gets the browser locale.

Reply:
For auto-detection I found this method to make all en-XX locales to be translated into the 2 digit version .e.g. "en,ja,es, etc...". Also if we do not have a local that matches the users preference then it will fall back to the "en" default english translations.

```
const i18n = new VueI18n({
  locale: navigator.language.split('-')[0],
  fallbackLocale: 'en',
  messages,
});
```
reference vue-i18n issue 200:
https://github.com/kazupon/vue-i18n/issues/220